MobileWallet/Common/Extensions/UIViewController.swift

```diff
@@ -88,6 +88,7 @@ extension UIViewController {
             let navBar = navController.navigationBar
 
             navBar.barTintColor = Theme.shared.colors.navigationBarBackground
+            navBar.titleTextAttributes = [NSAttributedString.Key.font: UIFont(name: "Avenir-Heavy", size: 14)!]
```

Comment:
Create a var in the fonts struct inside the theme file and reference it from there.